# corescripts
This repository contains scripts used for performing variuos processing steps and analysis presented in the CSBJ paper.

1)"Database_construction.html": contains a detailed explanation of the steps used to generate the enriched database.

2)"multiqc_stats.py": script used to generated qc stats from multiqc reports.

3)"process_counts_K2.py": script used to generate specific taxon counts of interest from Kraken2 reports for databases comparison.

4)"process_counts_Bracken.py": script used to generate the corresponding count table from the Bracken reports.

5)"get_bracken_taxa_table.py": script to generate taxonomy table from bracken count table using NCBI Taxonomy.

6)"DB_comparative.Rmd": script for comparison of genome databases and figures creation.

7)"Microbiome_analysis_genus.Rmd": script for analysis at genus level and figures creation.

8)"Microbiome_analysis_sp.Rmd": script for analysis at species level and figures creation.
