# corescripts
This repository contains scripts used for performing variuos processing steps and analysis presented in the CSBJ paper.

# Scripts used in Databases construction: 
filterN_seqs.sh, accession2taxid.R and check_seq2taxid_accession.py; A detailed description can be found in the supplementary material of the publication.

# multiqc_stats.py
Script used to generated some qc stats from multiqc reports.

# process_counts_K2.py
Script used to generate specific taxon counts of interest from Kraken2 reports for databases comparison.

# process_counts_Bracken.py
Script used to generate the corresponding count table from the Bracken reports.

# get_bracken_taxa_table.py
Script to generate taxonomy table from bracken count table using NCBI Taxonomy.

# DB_comparative.Rmd
Script for comparison of genome databases and figures creation.

# Microbiome_analysis_genus.Rmd
Script for analysis at genus level and figures creation.

# Microbiome_analysis_sp.Rmd
Script for analysis at species level and figures creation.
